config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'ticket_desks#index'
+  resources :cinema_halls
+  resources :ticket_desks
+  resources :tickets
+  resources :reservations
```

Comment:
You have reservations and tickets so it's a great moment to create nested paths. Consider (when you will have associations) nested routing like:
``` 
resources :reservations do 
  resources :tickets
end
```

Reply:
Is it a bad idea to create something like this in the future:
```
resources :users do
  resources :reservations do 
    resources :tickets
  end
end
```
and I should consider another approach?

Reply:
No, definitely no! :) This structure of routes looks really cool and I believe that you will be able to create a really good and robust API with this approach. Good idea! :)